lib/rubygems/gemcutter_utilities.rb

```diff
@@ -267,6 +269,28 @@ def get_scope_params(scope)
     scope_params
   end
 
+  def get_mfa_params(email, password)
+    return {} unless self.host == Gem::DEFAULT_HOST
+
+    mfa_level = get_user_mfa_level(email, password)
+    params = {}
+    if mfa_level == "ui_only" || mfa_level == "ui_and_gem_signin"
+      selected = ask "Would you like to enable MFA for this key? [Y/n]"
```

Comment:
Asking for `Y/n` value is happening already in `get_scope_params`. Do you think it would be possible to refactor this logic into "helper" private method within this class (`ask_yes_no`)?

Reply:
In `get_scope_params` we're asking for `N/y` (as in No is the default) but in this method we're asking `Y/n` (as in Yes is the default). So they're slightly different. I didn't see a super clean way of merging the two into one method without having conditionals that made it more confusing than being separate

Reply:
There is actually `ask_yes_no` method in `user_interaction.rb`. Can we use this one to keep it simple? It solves also another problems like fail hard on non-TTY frontend. I suggest to use that one and provide no default. That way it will fail as expected when user input is not available. Feel free to update the message to make it clear this is highly recommended to enable.

We should update the other (previously added) ask for input in this class as well to use the `ask_yes_no` method.